src/lambdas/get-app-theme-color.lambda.ts

```diff
@@ -9,12 +9,22 @@ enum Basic {
   'five' = `#29cc7f`,
 }
 
-enum Halloween {
-  'one' = `#ebedf0`,
-  'two' = `#ffee4a`,
-  'three' = `#ffc500`,
-  'four' = `#fe9601`,
-  'five' = `#d47901`,
+// enum Halloween {
+//   'one' = `#ebedf0`,
+//   'two' = `#ffee4a`,
+//   'three' = `#ffc500`,
+//   'four' = `#fe9601`,
+//   'five' = `#d47901`,
+// }
+
+enum Christmas {
+  'one' = `#edf8e9`,
+  'two' = `#c7e9c0`,
+  'three' = `#74c476`,
+  'four' = `#238b45`,
+  'five' = `#005a32`,
+  'six' = `#b01b2e`,
+  'seven' = `#ffffff`,
```

Comment:
Only five colors please xD (just like `Halloween` or `Basic`

Reply:
How about this
<img width="1453" alt="image" src="https://github.com/user-attachments/assets/3df36c6e-b20f-4fdd-9673-d70c418024bd">